src/openrct2/world/TileElement.h

```diff
@@ -380,13 +380,18 @@ struct TrackElement : TileElementBase
     ride_id_t RideIndex;
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wunused-private-field"
-    uint8_t pad[2];
+    // uint8_t pad[2];
+    uint8_t RideType;
```

Comment:
Put this above the pragma - it’s no longer an unused field. Only leave the `uint8_t pad;` in it.